### PR TITLE
@craigspaeth Adds FB :zap: IA flag

### PR DESCRIPTION
**Pull request by kanaabe**
Closes https://github.com/artsy/positron/issues/716

The xapp stuff is to handle tests better, there was this occasional test that randomly couldn't access artsyXapp.token so just set it to '' if it doesn't have it.

### CODE REVIEW

**Comment by kanaabe** [comment]
I wish GH would acknowledge emojis in headers 😿

File: api/apps/articles/model/save.coffee [hunk 1]
```diff
@@ -23,7 +23,7 @@ cloneDeep = require 'lodash.clonedeep'
 EMBEDLY_KEY, FORCE_URL, ARTSY_EDITORIAL_ID, SECURE_IMAGES_URL } = process.env
 sailthru = require('sailthru-client').createSailthruClient(SAILTHRU_KEY,SAILTHRU_SECRET)
 { crop } = require('embedly-view-helpers')(EMBEDLY_KEY)
-artsyXapp = require('artsy-xapp')
+artsyXapp = require('artsy-xapp').token or ''
```
Comment (craigspaeth) [inline api/apps/articles/model/save.coffee]: I think ideally we'd [rewire](https://github.com/jhnns/rewire) `artsyXapp` instead here, but I don't see any big immediate issues with this.

**Comment by craigspaeth** [comment]
Easy peasy! I too wish emojis worked in the headers 😭  Minor comment, feel free to follow up PR about it or not.

File: api/apps/articles/model/save.coffee (hunk 1 above)
Comment (kanaabe) [inline api/apps/articles/model/save.coffee]: Yeah, do you mean like [this?](https://github.com/artsy/positron/blob/master/api/apps/articles/test/model/save.coffee#L25). I'm probably rewiring it incorrectly, but this works intermittently

Comment (craigspaeth) [inline api/apps/articles/model/save.coffee]: Shoot, yeah 😛  I'll have to investigate more to see why that's failing intermittently ¯\_(ツ)_/¯

**Pull request merged**